lms/static/js/groups/models/cohort_discussions.js

```diff
@@ -0,0 +1,14 @@
+var edx = edx || {};
+
+(function(Backbone) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.DiscussionTopicsModel = Backbone.Model.extend({
```

Comment:
This isn't really a model of discussion topics. It is a model of which discussion topics are cohorted. Maybe ``DiscussionTopicsSettingsModel``?